tagstudio/src/qt/flowlayout.py

```diff
@@ -23,101 +23,149 @@
 
 
 class FlowWidget(QWidget):
+    """
+    From what I understand:
+    A generic Qt window.
+    Please fix  my description
+    """
     def __init__(self, parent=None) -> None:
         super().__init__(parent)
         self.ignore_size: bool = False
 
 
 class FlowLayout(QLayout):
-    def __init__(self, parent=None):
+    """A generic Qt layout + the ability to keep track of the items in the layout."""
+    def __init__(self, parent=None) -> None:
+        # Initialize the parent class
         super().__init__(parent)
 
+        # In case parent is set
         if parent is not None:
+            # Set margins to 0.
             self.setContentsMargins(QMargins(0, 0, 0, 0))
 
-        self._item_list = []
-        self.grid_efficiency = False
+        # Initialize the items list to nothing.
+        self._item_list: list = []
+        # Initialize grid efficiency to "False"
+        self.grid_efficiency: bool = False
 
     def __del__(self):
         item = self.takeAt(0)
         while item:
             item = self.takeAt(0)
 
-    def addItem(self, item):
+    def addItem(self, item) -> None:
+        """Add a given item to the list of items inside the layout."""
         self._item_list.append(item)
 
-    def count(self):
+    def count(self) -> int:
+        """Returns the amount of items in the layout"""
```

Comment:
[PEP-257](https://peps.python.org/pep-0257/#one-line-docstrings)  (Docstring Conventions) says:

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

Reply:
Sorry, could you maybe um explain it a little further? I am having trouble understanding you. Thank you

Reply:
Huh?